Scripts/IridisEnv/examples_cpu_job.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
Suggest that this script specifically says which example it works with, eg

```suggestion
# This script submits an Iridis5 batch job for the example in
# Examples/KelvinHelmholtz/SingleFluidIdealRandomMPIHDF5
# Note the seed after the ./main command would need removing for non-random cases.
```

This is mentioned in `workflows.md` but may be usefully cross-ref'd here.